src/transforms/filter.jl

```diff
@@ -46,11 +64,33 @@ Drop all rows with missing values in table.
     DropMissing([col₁, col₂, ..., colₙ])
     DropMissing((col₁, col₂, ..., colₙ))
 
-Drop all rows with missing values in selects columns `col₁`, `col₂`, ..., `colₙ`.
+Drop all rows with missing values in selects columns `col₁`, `col₂`, ..., `colₙ`.  
+The `col` arguments must be the same type and the accepted types 
+for `col` arguments are: `Integer`, `Symbol` or `String`.
```

Comment:
```suggestion
```

This is a technical detail that is not useful to end users. They will just try whatever combination and will figure out by themselves that it works with Symbol, String, etc.